docs/how-to/understanding-break-the-monolith.md

```diff
@@ -0,0 +1,46 @@
+---
+id: understanding-break-the-monolith
+title: Understanding Break The Monolith
+sidebar_label: Understanding Break The Monolith
+slug: /understanding-break-the-monolith
+---
+
+# Understanding Break The Monolith
+
+[Break The Monolith](/break-the-monolith) is a comprehensive tool for transforming monolithic architectures into modern, scalable, and maintainable microservices.
+
+This page explains the technical workings of this feature to help you understand how it operates under the hood.
+
+## How "Break The Monolith" Works
+
+The "Break The Monolith" feature consists of several steps, each designed to help you smoothly transition your monolithic service into a set of microservices.
+
+### 1. Overview of Current Services
+
+First, the feature provides an **Overview** mode where you can see a detailed view of your project's current services, entities, fields, and their relationships. This helps you understand the existing structure before making any changes.
+
+Overview mode provides you with a service mesh visualization, showing the services in your project, the entities owned by each service, and the relations between the services.
+
+### 2. Redesign and Breaking Services
+
+In the Redesign mode, you can manually or with the assistance of AI, restructure your monolith or individual services. This includes the ability to:
+
+- Move entities between existing or new services.
+- Utilize the Intelligent AI Helper for suggestions on optimal microservice configurations.
+- Create tailored micro-services based on the suggested architecture.
+
+The AI Helper identifies logical groupings of entities based on their relationships and usage patterns. It suggests ways to cut relations between entities beyond group boundaries, transforming foreign keys into remote IDs where necessary.
```

Comment:
The transformation of foreign keys to IDs is done as part of the apply operation, not as part of the AI.